serverless-single-page-app-plugin/index.js

```diff
@@ -0,0 +1,151 @@
+'use strict';
```

Comment:
There is a duplication. The same file exists in a directory '.serverless_plugins'. Please keep one copy of this file.

Reply:
@cetrix13 
To my understanding I need to remove .serverless_plugins folder, as package.json has a mention of 
"serverless-single-page-app-plugin": "file:./serverless-single-page-app-plugin",. 
Could you confirm if I am correct here?

Reply:
Yes, you're right.